### DISCUSSION TITLE
Is Xamarin.Forms (still) supported?

### DISCUSSION THREAD

**Discussion by NicolasKlaas** [QuestPDF/QuestPDF#977] [Q&A] [answered]
I wanted to integrate QuestPDF into my Xamarin.Forms project before migrating it to MAUI. However, I am getting the error ‘Your runtime is currently not supported by QuestPDF.’ Note that Xamarin.Forms also targets .NET Standard 2.1. Is there any way to overcome this, perhaps by using an older version of QuestPDF?

The error: 
```
Your runtime is currently not supported by QuestPDF. Currently supported runtimes are: win-x86, win-x64, linux-x64, linux-arm64, linux-musl-x64, osx-x64, osx-arm64. 

It appears that your current operating system distribution may be outdated. For optimal compatibility, please consider updating it to a more recent version. ---> System.DllNotFoundException: QuestPdfSkia assembly:<unknown assembly> type:<unknown type> member:(null)
  at (wrapper managed-to-native) QuestPDF.Skia.SkNativeDependencyCompatibilityChecker+API.check_compatibility_by_calculating_sum(int,int)
  at QuestPDF.Skia.SkNativeDependencyCompatibilityChecker.
```

**Comment by NicolasKlaas** [comment]
After downgrading to the `2023.12.X` versions, and reading this comment: https://github.com/QuestPDF/QuestPDF/discussions/931#discussioncomment-9814431, I reckon that it is no longer possible due to 'significant internal redesign' in the `2024` release.

**Reply by MarcinZiabek** [reply, marked as answer]
It is mostly a matter of priorities rather than any technology limitations. Supporting MAUI requires creating additional native builds for Android and iOS (https://github.com/QuestPDF/QuestPDF.Native/blob/main/.github/workflows/build_questpdf_skia.yml). Depending on Community feedback and/or help, the support could be reintroduced 😄

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
